src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -1297,9 +1328,18 @@ private void addUpdatedDocIds(ImmutableList.Builder<Record> builder,
   }
 
   private String makeUpdatedDocsQuery(Checkpoint checkpoint) {
+    if (updatedDocumentsQuery.isEmpty()) {
+      return makeUpdatedDocsDefaultQuery(checkpoint);
+    } else {
+      return MessageFormat.format(updatedDocumentsQuery,
```

Comment:
This throws IllegalArgumentException, which isn't caught by the connector. That breaks the traversers, which only handle DfExceptions. There's a related problem in validateUpdatedDocumentsQuery, where it would stop the connector, I think.

Handling it during validation is trivial. Handling it here would require throwing a fake DfException, or changing the traversers. We could go with your original suggestion, and if the validation fails reset the field to empty so that we'll use the default query here.